uber/site_sections/groups.py

```diff
@@ -75,11 +75,26 @@ def unapprove(self, session, id, action, email):
         if action == 'waitlisted':
             group.status = c.WAITLISTED
         else:
+            message = 'Group declined'
             for attendee in group.attendees:
-                session.delete(attendee)
+                if attendee.is_unassigned or not convert:
+                    session.delete(attendee)
+                else:
+                    message = 'Group declined and emails sent to attendees'
+                    attendee.paid = c.NOT_PAID
+                    attendee.badge_status = c.NEW_STATUS
+                    attendee.ribbon = c.NO_RIBBON
+                    send_email(c.REGDESK_EMAIL, attendee.email, 'Do you still want to come to {EVENT_NAME}?',
+                               render('emails/dealers/badge_converted.html', {
+                                   'attendee': attendee,
+                                   'group': group
+                               }), model=attendee)
+                    group.attendees.remove(attendee)
+                    group.leader = None
```

Comment:
If we wanted to be paranoid about AWS being down (which happens pretty rarely) we'd wrap the ``send_email`` call in a try/except block and then add ``" (but the automated email did not get sent)"`` to ``message``.